content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -153,6 +153,7 @@ section refers to several key workload enablers and how they map to Windows.
   * `emptyDir` volumes
   * Named pipe host mounts
   * Resource limits
+  * OS field: This value should be set to `windows` to indicate that the current pod is a Windows pod.
```

Comment:
We should either add a note around here saying this functionality is in alpha or link to another page/section that does so.